contrib/epee/include/string_tools.h

```diff
@@ -47,12 +48,12 @@ namespace epee
 {
 namespace string_tools
 {
-  //----------------------------------------------------------------------------
```

Comment:
We have historically refrained from changing whitespace comments (or in this case decorator) such as this.

Reply:
Returned all the non-code changes to previous state.